chacra/async/rpm.py

```diff
@@ -73,16 +73,24 @@ def create_rpm_repo(repo_id):
         except OSError:
             logger.exception('could not symlink')
 
-    for d in repo_dirs:
-        # this prevents RPM packages that are larger than 2GB (!!!) from
-        # causing the database to fail to store the size and subsequently make
-        # the package uninstallable. Ideally, this types of flag options should
-        # be configurable
-        subprocess.check_call(['createrepo', '--no-database', d])
+    _createrepo(paths['absolute'], repo_dirs, repo.distro)
 
     logger.info("finished processing repository: %s", repo)
     repo.is_updating = False
     models.commit()
     timer.stop()
     counter += 1
     post_ready(repo)
+
+
+def _createrepo(base_path, repo_dirs, distro):
+    if distro in ['opensuse', 'sle']:
```

Comment:
it would be more robust if you guard against case combinations. This would break if someone creates a package with OpenSuse or SLE.

Reply:
Done (also added a test for that)

Reply:
Hm, interesting, we do not use other labels in teuthology rather than 'opensuse' or 'sle', and it should be always lowercase. From other size from generalization point of view we need to include all possible cases for suse base repos, like: opensuse, leap, tumbleweed, sle, sles, etc. which we can not predict at this moment all possible names.